### PR TITLE
Smoke Tests

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
Created 3 basic tests:

1. A form renders
2. An input field renders
3. The input can be typed into

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest --watch"
```
Comment (mdwiltfong) [inline package.json]: Uh oh! I think  `--watch` is causing the GH action to stall

Comment (mdwiltfong) [reply package.json]: The test job was actually running for 1 hour. I had to cancel it!

Comment (Jamesllllllllll) [reply package.json]: Oh crap! Fixed.

**Review by mdwiltfong — APPROVED** [review]
LGTM!

**Pull request merged**